autometa/taxonomy/get_lca_taxa.py

```diff
@@ -0,0 +1,127 @@
+#!/usr/bin/env python
+
+import os
+import glob
+import logging
+import pyreadr
```

Comment:
Is this necessary?

Can you just convert the RDS file into a TSV or CSV then use it with this script?